prog/test/vm_group.rb

```diff
@@ -37,21 +37,24 @@ def self.assemble(storage_encrypted: true)
     vm1_s = Prog::Vm::Nexus.assemble(
       keypair_1.public_key, project.id,
       private_subnet_id: subnet1_s.id,
-      storage_encrypted: frame["storage_encrypted"],
+      storage_volumes: [
+        {encrypted: frame["storage_encrypted"]},
```

Comment:
seems like the `[]` returning nil behavior may not be ideal here (false, turning encryption off, in event of a typo in `storage_encrypted`)